eng/Versions.props

```diff
@@ -18,7 +18,7 @@
     <!-- F# Version components -->
     <FSMajorVersion>10</FSMajorVersion>
     <FSMinorVersion>0</FSMinorVersion>
-    <FSBuildVersion>100</FSBuildVersion>
+    <FSBuildVersion>200</FSBuildVersion>
     <FSRevisionVersion>0</FSRevisionVersion>
```

Comment:
`main` already maps to .NET11 in the VMR (out which .NET is shipped).
So this should rather map to the first preview of .NET 11 (including the 2 branding elements above, I guess preview1).

`release/dev18.0` would be then used for the minor increments that map into .NET10.xxx of the VMR.

Reply:
Yes, you're right, d18 should be bumped to 200 then